cluster-agent/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Apache License is required by charts to be included in the community repo. https://github.com/IBM/charts/blob/master/GUIDELINES.md#license
This license is for the helm chart.
Any product licenses can go in a separate LICENSES folder.